src/app/modules/auth/auth.utils.ts

```diff
@@ -63,3 +65,7 @@ export const getUserIdFromSession = (
 ): string | undefined => {
   return session?.user?._id as string | undefined
 }
+
+export const isCronPaymentAuthValid = (header: IncomingHttpHeaders) => {
+  return header['x-formsg-cron-payment-secret'] === cronPaymentConfig.apiSecret
```

Comment:
/toDiscuss (potentiallyNit/canProbablyDefer): since the API is public, and just protected by secret auth, we might want to store the shared secret hashed in the config, and do a hash comparison here. So if a malicious user ever gets access to our config key, they won't be able exit silently, while still be able to invoke the APIs.